tools/setup.ps1

```diff
@@ -0,0 +1,38 @@
+
+$ROOT = ( get-item $PSScriptRoot ).Directory.parent.parent.FullName
+
+# Stop when error occurs
+$ErrorActionPreference = "Stop"
+
+$ChocoInstalled = $false
+
+function local:Status([String] $Message){
+  Write-Host $Message -ForegroundColor DarkMagenta
+}
+
+
+Status("1. Install nvm")
+if (Get-Command choco.exe -ErrorAction SilentlyContinue) {
+  $ChocoInstalled = $true
+  choco install nvm
+} else {
+  Install-Module -Name nvm
```

Comment:
It's awesome if this works, I am just thinking we should get some confirmation that a default Win10 install actually can successfully install `nvm` in this manner. I suppose AppVeyor should give us that insight to some degree.